freetextresponse/private/view.js

```diff
@@ -4,46 +4,125 @@ function FreeTextResponseView(runtime, element) {
     var $ = window.jQuery;
     var $element = $(element);
     var buttonSubmit = $element.find('.check.Submit');
-    var textareaParent = $element.find('.student_answer').parent();
+    var textareaParent = $element.find('.textarea').parent();
     var usedAttemptsFeedback = $element.find('.action .used-attempts-feedback');
     var problemProgress = $element.find('.problem-progress');
     var submitParent = $element.find('.Submit').parent();
-    var wordCountError = $element.find('.word-count-error');
     var submissionReceivedMessage = $element.find('.submission-received');
     var url = runtime.handlerUrl(element, 'submit');
     var userInputClass = 'user-input';
+    var buttonSave = $element.find('.save');
+    var saveParent = $element.find('.save').parent();
+    var urlSave = runtime.handlerUrl(element, 'save_reponse');
```

Comment:
Maybe make this stand out more by moving it to either the top or bottom of this stanza, instead of nesting it into the middle of CSS selectors (since it's an ajax handler).